src/test/java/com/commercetools/dataimport/ImportJobIntegrationTest.java

```diff
@@ -11,13 +11,11 @@
     @Test
     public void runs() {
         assertThat(fetchTotalCategories()).as("Categories are imported").isEqualTo(131);
-        assertThat(fetchTotalProducts()).as("Products are imported").isEqualTo(2);
-        assertThat(fetchTotalChannels()).as("Channels are imported").isEqualTo(18);
         assertThat(fetchTotalTypes("order")).as("Order types are imported").isEqualTo(1);
         assertThat(fetchTotalTypes("customer")).as("Customer types are imported").isEqualTo(1);
         assertThat(fetchTotalTypes("channel")).as("Channel types are imported").isEqualTo(1);
-        assertThat(fetchTotalOrders()).as("Orders are imported").isEqualTo(2);
-        assertThat(fetchTotalInventory()).as("Inventory is imported").isEqualTo(4);
+// As the products are imported by the nodejs Library, the Order cant be tested, as the SKU is not available in the system.
+// assertThat(fetchTotalOrders()).as("Orders are imported").isEqualTo(2);
```

Comment:
Can we enable all the tests? All the import endpoints I implemented during the previous weeks (products, categories, project settings, etc.) were written for the sole purpose of not having any code untested in the java app, for easing up the maintenance as long as it stays alive. If we disable them here then all the purpose is lost.

To enable them we just have to split the `catalogImportFlow` into categories and the rest, so that you can enable the category import from the node.js app, while the rest is only enabled for tests.